html_css/v2/foundations/intro-to-html-css.md

```diff
@@ -0,0 +1,25 @@
+### Introduction
+So here it is: it's time to actually start _making things_. This section will teach you the basics of HTML and CSS, the two foundational building blocks of pretty much everything on the web.
+
+### Learning Outcomes
+* Get a basic overview of HTML, CSS and how they work together.
+
+### HTML and CSS
+HTML and CSS are two languages that work together to create everything that you see when you look at something on the internet. HTML is the raw data that a webpage is built out of. All the text, links, cards, lists and buttons are created in HTML. CSS is what adds _style_ to those plain elements. HTML is used to put information on a webpage and CSS is used to position that information, give it color, change the font and make it look great!
+
+If you want to get technical, HTML and CSS aren't _programming languages_ because they are only concerned with presenting information. They are not used to program any logic. JavaScript, which you will be learning in the next section, is a programming language because it's used to make webpages do things. There is quite a lot you can accomplish with just HTML and CSS however, and you will definitely need them both. The following lessons focus on giving you the tools you need to be successful once you get to the JavaScript content as you continue through our curriculum!
+
+### Assignment
+<div class="lesson-content__panel" markdown="1">
```

Comment:
The two links in this lesson I think should have better names than "this article".

The first one could be, "Read [HTML vs. CSS vs. Javascript](...) to get a quick overview of the relationship between HTML, CSS, and JavaScript."

The second could be, "[HTML, CSS, and JavaScript Explained for Beginners](...) goes into..."

(first time commenting on a range of lines in a PR)